test/TestUtilities/Test.Utility/compiler/resources/project2.csproj

```diff
@@ -4,7 +4,7 @@
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
```

Comment:
I don't think this necesarilly needs changed. 
it's part of the tests, and overriden most of the time anyways.